src/bond.py

```diff
@@ -63,3 +63,100 @@ def __init__(self):
             ),
         )
         self._add_typeparam(params)
+
+
+class Quartic(bond.Bond):
+    r"""Quartic bond potential.
+
+    `Quartic` specifies a quartic potential between the two particles in
+    the simulation state with:
+
+    .. math::
+        \begin{eqnarray*}
+        U(r) = & k [(r - r_{0}) - b_{1}][(r - r_{0}) - b_{2}](r - r_{0})^2
+                  + U_{0} + U_{\rm WCA}(r)      & \text{when } r < r_0\\
+             = & U_0                            & \text{when } r\ge r_0
+        \end{eqnarray*}
```

Comment:
I think you should include Delta in the equations explicitly. Is this right?
```suggestion
        :nowrap:

        \begin{eqnarray*}
        U(r) &= k (r - \Delta - r_0 - b_1)(r - \Delta - r_0 - b_2)(r - \Delta - r_0)^2
                  + U_0 + U_{\rm WCA}(r), & r < r_0 + \Delta \\
             &= U_0, & r \ge r_0 + \Delta
        \end{eqnarray*}
```
You can add an internal line break if the equation gets too wide.